front/lib/amplitude/back/generated/index.ts

```diff
@@ -98,6 +98,13 @@ export interface DataSourceCreatedProperties {
   workspaceId: string;
   workspaceName: string;
 }
+export interface DataSourceUpdatedProperties {
```

Comment:
@PopDaph did you write this code or did you generate it from Amplitude/ampli ?

Reply:
I wrote it 😬

Reply:
@PopDaph that's on me, I should have documented it.
I will invite you to Amplitude and write a little doc.